packages/migrate/migrations/sveltekit-2/migrate.js

```diff
@@ -0,0 +1,112 @@
+import fs from 'node:fs';
+import { Project, Node } from 'ts-morph';
+import { log_migration, log_on_ts_modification, update_pkg } from '../../utils.js';
+
+export function update_pkg_json() {
+	fs.writeFileSync(
+		'package.json',
+		update_pkg_json_content(fs.readFileSync('package.json', 'utf8'))
+	);
+}
+
+/**
+ * @param {string} content
+ */
+export function update_pkg_json_content(content) {
+	return update_pkg(content, [
+		// All other bumps are done as part of the Svelte 4 migration
+		['@sveltejs/kit', '^2.0.0'],
+		['vite', '^5.0.0'],
+		[
+			'@sveltejs/vite-plugin-svelte',
+			'^3.0.0',
+			' (vite-plugin-svelte is a peer dependency of SvelteKit now)',
+			'devDependencies'
+		]
+		// TODO bump vitest? others?
+	]);
+}
+
+export function update_tsconfig() {
+	fs.writeFileSync(
+		'tsconfig.json',
+		update_tsconfig_content(fs.readFileSync('tsconfig.json', 'utf8'))
+	);
+}
+
+/** @param {string} content */
+export function update_tsconfig_content(content) {
```

Comment:
can we also make sure typescript is on v5 and that `moduleResolution` is `bundler` for applications and `NodeNext` or `Node16` for libraries?